sanitychecks.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2017 Tuenti Technologies S.L. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/achanda/go-sysctl"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+var (
+	ephemeralPortsRange *EphemeralPortsRange
+)
+
+type EphemeralPortsRange struct {
+	check    bool
+	LowPort  int32
+	HighPort int32
+}
+
+// Check if a given port is in the ephemeral ports range
+func (r EphemeralPortsRange) Contains(p int32) bool {
+	return (p >= r.LowPort) && (p <= r.HighPort)
+}
+
+// Provide the range as a String
+func (r EphemeralPortsRange) String() string {
+	return fmt.Sprintf("%d - %d", r.LowPort, r.HighPort)
+}
+
+// Retrieve the data from sysctl and return the values, disable the check if unsuccessful and log
+func initEphemeralPortsRangeCheck() *EphemeralPortsRange {
+	var l, h int32
+	r, err := sysctl.Get("net.ipv4.ip_local_port_range")
+	if err != nil {
+		log.Fatalf("Error reading from sysctl: %s, ephemeral ports range checks will be skipped", err)
```

Comment:
Fatalf is like Print and os.Exit(-1), I don't think we want to end here :) use just Print.